### PR TITLE
Update docs to reflect redis upstream latency is in micros only

### PR DESCRIPTION
**Pull request by FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg**
Commit Message: Update docs to reflect redis upstream latency is in micros only.
Additional Description: Currently docs say that you can use `latency_in_micros` to choose millis or micros; this doesn't make as much sense for Redis, but it also breaks convention we use with HTTP. I will consider fixing this a todo.
Risk Level: None
Testing: None
Docs Changes: Yees
Release Notes: None

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst [hunk 1]
```diff
@@ -48,8 +48,7 @@ Per command statistics
 ----------------------
 
 The Redis filter will gather statistics for commands in the
-*redis.<stat_prefix>.command.<command>.* namespace. By default latency stats are in milliseconds and can be
-changed to microseconds by setting the configuration parameter :ref:`latency_in_micros <envoy_v3_api_field_extensions.filters.network.redis_proxy.v3.RedisProxy.latency_in_micros>` to true.
+*redis.<stat_prefix>.command.<command>.* namespace. The latency stats are in microseconds.
```
Comment (ramaraochavali) [inline docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: Are you sure default is microseconds? Quick read of code indicates it is milliseconds and I remember that is how implemented. Proto doc also indicates the same https://github.com/envoyproxy/envoy/blob/549164c42cae84b59154ca4c36009e408aa10b52/generated_api_shadow/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto#L192 - Has it changed recently?

Comment (aviramha) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: All of the stats are in milliseconds, besides upstream statistics which are microseconds.

Comment (ramaraochavali) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: > besides upstream statistics which are microseconds.
which upstream stats?

Comment (aviramha) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: image

Comment (ramaraochavali) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: I see. Makes sense now. So we need clarify it just applies to upstream command level stats and keep the existing description for filter level stats as is?

Comment (aviramha) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: I believe so, yes.

Comment (FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: That's correct. I need to make the millis be the default, but I have a I bunch of other work to do first.

Comment (mattklein123) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: Can we do a clarification here and we can merge? Thank you.

/wait

Comment (FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg) [reply docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst:51]: I have updated the docs and put this under the setting sections.

**Pull request merged**